kernel: bump 5.10 to 5.10.115

graysky2 opened the pull request:
Removed upstreamed:
  backport-5.10/850-v5.17-0004-PCI-aardvark-Clear-all-MSIs-at-setup.patch
  pending-5.10/850-0002-PCI-aardvark-Fix-reading-MSI-interrupt-number.patch

All other patches automatically rebased.

Build system: x86_64
Build-tested: bcm2711/RPi4B

Signed-off-by: John Audia <therealgraysky@proton.me>

graysky2 committed:
kernel: bump 5.10 to 5.10.114

All patches automatically rebased.

Build system: x86_64
Build-tested: bcm2711/RPi4B
Run-tested: bcm2711/RPi4B

Signed-off-by: John Audia <therealgraysky@proton.me>

xdarklight commented:
I am hoping that someone from #8833 can provide their Tested-by.
This kernel update is supposed to fix #8833 as it backports https://git.kernel.org/pub/scm/linux/kernel/git/torvalds/linux.git/commit/?id=71cffebf6358a7f5031f5b208bbdc1cb4db6e539

graysky2 committed:
kernel: bump 5.10 to 5.10.115

Removed upstreamed:
  backport-5.10/850-v5.17-0004-PCI-aardvark-Clear-all-MSIs-at-setup.patch
  pending-5.10/850-0002-PCI-aardvark-Fix-reading-MSI-interrupt-number.patch

All other patches automatically rebased.

Build system: x86_64
Build-tested: bcm2711/RPi4B

Signed-off-by: John Audia <therealgraysky@proton.me>

Borromini commented:
FWIW: tested on 22.03 HEAD.

Compile-tested: ath79/generic, octeon/generic, ramips/mt76{20,21,x8}, realtek/rtl{838,930}x, x86/64
Run-tested: ramips/mt7621, realtek/rtl838x.

Tested-by: Stijn Segers <foss@volatilesystems.org>

hauke commented:
Thank you for your patches, I applied them to master in 8592df67f40b3afdee68e36dc3820187ec0f98fc and b754b0c721131005efa7127151088e9c23dc9053.
